### PR TITLE
LPS-161358 Add button to hide sidebars

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:relevant

**Review by p2kmgcl — COMMENTED** [review]
I just added a tiny suggestion, I am going to test it manually and remove the review label :3

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/Sidebar.js [hunk 1]
```diff
@@ -78,8 +78,10 @@ export default function Sidebar() {
 	const store = useSelector((state) => state);
 
 	const panels = useSelector(selectAvailablePanels(config.panels));
-	const sidebarOpen = store.sidebar.open;
-	const itemConfigurationOpen = store.sidebar.itemConfigurationOpen;
+	const sidebarHidden = store.sidebar.hidden;
+	const sidebarOpen = store.sidebar.open && !sidebarHidden;
+	const itemConfigurationOpen =
+		store.sidebar.itemConfigurationOpen && !sidebarHidden;
```
[the remark this answers is not in the document]
Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/Sidebar.js:84]: True, I'll change that :D

**Comment by victorg1991** [comment]
ci:stop

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
Done @p2kmgcl  :D

**Comment by ozysouza** [comment]
Please see https://github.com/brianchandotcom/liferay-portal/pull/122126

**Pull request closed without merging**